sml_small/editing/thousand_pounds/thousand_pounds.py

```diff
@@ -103,81 +172,37 @@ def determine_tpc_marker(do_adjustment: bool) -> str:
 def determine_predictive_value(
     predictive: Optional[float], auxiliary: Optional[float]
 ) -> float:
-    if predictive:
-        validate_number("predictive", predictive)
-        return float(predictive)
+    if predictive or predictive == 0:
```

Comment:
Could we just make this if predictive >= 0 ?

Reply:
that can cause issues as it wont necessarily capture interpretable data types, e.g "10" may come back as False in that case.  the above just gets around python evaluating 0 as empty, similar to "", None, or []

Reply:
Thanks for the explanation - let's leave as is for this ticket, although I think probably we might be better off converting the input to a number format (Decimal) so that it is obvious we are dealing with only numeric types throughout the rest of the function and return Decimal. We can add a separate ticket to do this if you agree?

Reply:
also just realised predictive can still be None at this point as we could be using auxiliary instead, if >=0 will cause a typeError in that instance